tasks/variables.yml

```diff
@@ -6,7 +6,11 @@
 
 - name: Include OS-specific variables (RedHat).
   include_vars: "{{ ansible_os_family }}-{{ ansible_distribution_version.split('.')[0] }}.yml"
-  when: ansible_os_family == 'RedHat'
+  when: ansible_os_family == 'RedHat' and ansible_distribution != 'Fedora'
```

Comment:
Can you format this as:

```yaml
when:
  - ansible_os_family == 'RedHat'
  - ansible_distribution != 'Fedora'
```

Just makes it easier to read (IMO) and is the pattern I use elsewhere for ANDs

Reply:
Done; I agree with this style.